app/web/src/components/PageTitle.tsx

```diff
@@ -8,8 +8,19 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-export default function PageTitle(props: TypographyProps) {
+type PageTitleProps = TypographyProps & {
+  className?: string;
+};
+
+export default function PageTitle({ className, ...props }: PageTitleProps) {
   const classes = useStyles();
 
-  return <Typography {...props} classes={{ h1: classes.root }} variant="h1" />;
+  return (
+    <Typography
+      {...props}
+      className={className}
+      classes={{ h1: classes.root }}
```

Comment:
I know this isn't your code, but we're moving away from using `classes` since we eventually want to move to MUIv5 and Emotion, which doesn't support it (it's not necessary). Instead can you do `className={classNames(className, classes.root)}` (using the `classNames` package which we already use to combine classes etc.)

Reply:
I removed it in last push. In this case I didn't need to merge two classes because extra styling of the pageTitle was not needed anymore.